chrome/manifest.json

```diff
@@ -53,6 +53,9 @@
     "images/icon-13.jpg",
     "images/icon-16.jpg",
     "images/new_icon.gif",
+    "images/clock.png",
+    "images/archived.png",
+    "images/trello-icon-black.png",
```

Comment:
Trello uses an icon font, which would be nice, but since I only need a couple of icons I figured images would work. "Trello Icon Black" is a new icon for gmail, since I now need to show either a grey or green button depending on the page status.